pages/linux/xdg-settings.md

```diff
@@ -0,0 +1,24 @@
+# xdg-settings
+
+> Get various settings from the desktop environment.
+> More information: <https://portland.freedesktop.org/doc/xdg-settings.html>.
+
+- Print default web browser:
+
+`xdg-settings get default-web-browser`
+
+- Set default web browser to Firefox:
+
+`xdg-settings set default-web-browser firefox.desktop`
+
+- Set default mail URL scheme handler to evolution.desktop:
```

Comment:
Since you're also using `Firefox`, and not `firefox.desktop` in the previous example:
```suggestion
- Set the default mail URL scheme handler to Evolution:
```